NeuroID/src/androidLib/java/com/neuroid/tracker/events/NIDIdentifyAllViews.kt

```diff
@@ -29,27 +29,31 @@ import org.json.JSONObject
 fun identifyAllViews(
     viewParent: ViewGroup,
     guid: String,
+    logger: NIDLogWrapper,
+    storeManager: NIDDataStoreManager,
```

Comment:
I don't see this being used anywhere?

Reply:
These are used. logger is used in place of NIDLogger and storeManager in place of getDataStoreInstance().

Reply:
Have to keep the storeManager mock for the tests to complete.